Sources/SecurityRequirement.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import ObjectMapper
+
+public struct SecurityRequirement: ImmutableMappable {
+    public let name: String
+    public let scopes: [String]
+
+    public init(map: Map) throws {
+        name = map.JSON.first!.key
```

Comment:
I don't want us to do any force-unwrapping. If the expected value isn't there, throw a DecodingError()
```
public init(map: Map) throws {
    let dictionary = try [String : [String]](map: map)
    guard let name = dictionary.keys.first, let scopes = dictionary[name] else {
        throw DecodingError()
    }

    self.name = name
    self.scopes = scopes
}
```